### PR TITLE
Update onboarding and about app copy text and font size

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Link: https://phabricator.wikimedia.org/T258132

### Problem Statement

_No problem_

### Solution

_as title_

### Note

high priority

### CODE REVIEW

File: style/aboutwikipedia.less [hunk 1]
```diff
@@ -43,8 +43,8 @@
 
 		.title {
 			font-weight: bold;
-			font-size: 22px;
-			line-height: 26px;
+			font-size: 20px;
+			line-height: 23.6px;
```
Comment (medied) [inline style/aboutwikipedia.less:47]: Assuming that .6 is not a typo

Comment (hueitan) [reply style/aboutwikipedia.less:47]: nope, 

_unfortunately this is not on the ticket but the value is given through direct message_

**Comment by medied** [comment]
@jpita ready for testing

**Comment by hueitan** [comment]
Feel free to test and move ticket after the merge

**Pull request merged**